src/app/vm/vm-creation/containers/vm-creation.container.ts

```diff
@@ -102,25 +104,29 @@ export class VmCreationContainerComponent implements OnInit {
   readonly account$ = this.store.pipe(select(fromAuth.getUserAccount));
   readonly zones$ = this.store.pipe(select(fromZones.selectAll));
   readonly sshKeyPairs$ = this.store.pipe(select(fromSshKeys.selectSshKeysForAccount));
+  public isDiskOfferingAvailableByResources$;
+  public minSize: number;
 
   constructor(
     private store: Store<State>,
     private authService: AuthService,
     private dialogRef: MatDialogRef<VmCreationContainerComponent>,
   ) {
+    this.store.dispatch(new templateActions.LoadTemplatesRequest());
     this.store.dispatch(new securityGroupActions.LoadSecurityGroupRequest());
     this.store.dispatch(new zoneActions.LoadZonesRequest());
-    this.store.dispatch(new templateActions.LoadTemplatesRequest());
     this.store.dispatch(new sshKeyActions.LoadSshKeyRequest());
     this.store.dispatch(new diskOfferingActions.LoadOfferingsRequest());
     this.store.dispatch(new affinityGroupActions.LoadAffinityGroupsRequest());
     this.store.dispatch(new serviceOfferingActions.LoadOfferingsRequest());
     this.store.dispatch(
       new accountTagsActions.LoadAccountTagsRequest({ resourcetype: accountResourceType }),
     );
-
+    this.minSize = this.authService.getCustomDiskOfferingMinSize();
```

Comment:
We will need to refactor capabilities into ngrx

Reply:
We will discuss it on the next meet